lib/OpenQA/Utils.pm

```diff
@@ -399,5 +400,15 @@ sub check_download_whitelist {
     return ();
 }
 
+sub notify_workers {
+    my $ipc = OpenQA::IPC->ipc;
+
+    my $con = $ipc->{bus}->get_connection;
+    my $msg = $con->make_method_call_message("org.opensuse.openqa.Scheduler", "/Scheduler", "org.opensuse.openqa.Scheduler", "emit_jobs_available");
```

Comment:
the problem is that if you do it with the high level API, it will call intro Introspect of the object to query the parameters. And this call will block. So I had to use the low level API